src/plumbing/progress.rs

```diff
@@ -408,10 +408,6 @@ static GIT_CONFIG: &[Record] = &[
         config: "http.sslCipherList",
         usage: NotPlanned { reason: "on demand" }
     },
-    Record {
```

Comment:
It's a good rule of thumb that whenever something is removed here, it should be added as a constant.

Reply:
I added the constants, but I am not sure if I should link it with the environment variable: GIT_SSL_NO_VERIFY, as I didn't add support for it and it is the negation of sslVerify.

Reply:
Yes, that's a great point!

Keys can have an environment variable attached to it [using this method](https://github.com/Byron/gitoxide/blob/345712dcdfddcccc630bbfef2ed4f461b21550d3/gix/src/config/tree/keys.rs#L64). These then needs to be [initialized here](https://github.com/Byron/gitoxide/blob/8434aab5fb6ce32be2bf3b20e38c28c780bd5db9/gix/src/config/cache/init.rs#L74). However, as it is a negation, we'd need add our own version of that variable as `gitoxide.http.sslNoVerify`, link it with (with a `description()`) `GIT_SSL_NO_VERIFY`, initialize it, [test it](https://github.com/Byron/gitoxide/blob/8434aab5fb6ce32be2bf3b20e38c28c780bd5db9/gix/tests/gix-init.rs#L23), and then let it override any value that `http.sslVerify` may have [just like git](https://github.com/git/git/blob/cfb8a6e9a93adbe81efca66e6110c9b4d2e57169/http.c#L1293-L1294).

Hooking up variables is quite involved, but that separation is needed to avoid global state (like environment variables) to sneak in. This way they are controlled and used exactly in one or two places in the codebase, while being introspectable via `gix config` and controllable (as one can turn off certain kinds of environment variables from being read).